charts/trino/templates/deployment-coordinator.yaml

```diff
@@ -68,6 +68,7 @@ spec:
         {{- end }}
       imagePullSecrets:
         {{- toYaml .Values.imagePullSecrets | nindent 8 }}
+      terminationGracePeriodSeconds: {{ .Values.coordinator.terminationGracePeriodSeconds | default 30 }}
```

Comment:
Remove the default, as it's in `values.yaml` now. Having defaults in two different places adds a risk of them getting out of sync, which can be confusing. This property should be either required (causing a failure if someone removes it from `values.yaml`), or optional, placed in a conditional block here so it doesn't render. I'd leave it required.

Reply:
Removed